### PR TITLE
Add NGC-Ready playbook and doc

### PR DESCRIPTION
**Pull request by dholt**
Also adds default config to render human-readable Ansible output.

### CODE REVIEW

File: docs/ngc-ready.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+This process will install the latest NVIDIA GPU Drivers, and Docker with the NVIDIA container runtime.
+
+```sh
+# Install NGC-Ready software stack
+# where:
+#   <ssh-user>: SSH username to reach NGC-Ready server
+#   <ip-of-host>: IP of NGC-Ready server, or localhost. The trailing comma is required
+# If SSH requires a password, add: -k
+# If sudo requires a password, add: -K
+ansible-playbook -u <ssh-user> -i <ip-of-host>, playbooks/ngc-ready.yml
```
Comment (michael-balint) [inline docs/ngc-ready.md]: `-i` should refer to inventory file, not ip. Might want to include `-l` if specifying limitation by host.

File: docs/ngc-ready.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+This process will test the functionality of the NGC-Ready server by running a functional test and two deep learning framework container tests from the NVIDIA Container Registry.
+
+```sh
+# Run NGC-Ready tests
+# where:
+#   <ssh-user>: SSH username to reach NGC-Ready server
+#   <ip-of-host>: IP of NGC-Ready server, or localhost. The trailing comma is required
+# If SSH requires a password, add: -k
+# If sudo requires a password, add: -K
+ansible-playbook -u <ssh-user> -i <ip-of-host>, playbooks/ngc-ready.yml --tags test
```
Comment (michael-balint) [inline docs/ngc-ready.md]: Same as above.

File: playbooks/ngc-ready.yml [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+        pull: yes
+      tags:
+        - test
+        - never
+    - name: test pytorch
+      docker_container:
+        name: pytorch
+        image: "nvcr.io/nvidia/pytorch:{{ ngc_container_tag }}"
+        working_dir: /opt/pytorch/examples/upstream/mnist
+        detach: no
```
Comment (michael-balint) [inline playbooks/ngc-ready.yml]: Would it hurt to add the following? Otherwise it won't work on virtual clusters, for example.

```
network_mode: host
```

**Comment by michael-balint** [comment]
Overall comment: the tests are useful, but isn't this just doing what we do with some of our other ansible playbooks? What is the value add?

File: docs/ngc-ready.md (hunk 1 above)
Comment (dholt) [reply docs/ngc-ready.md]: My impression was that this playbook would be used for one-off machines, so this was an attempt to simplify things and not force people to edit the inventory file. This is how I always use the standalone playbook as well.

File: playbooks/ngc-ready.yml (hunk 3 above)
Comment (dholt) [reply playbooks/ngc-ready.yml]: The tests don't require any networking so it's probably not necessary

Comment (michael-balint) [reply playbooks/ngc-ready.yml]: They do for some - for example the pytorch one fails if you can't download the MNIST dataset.

Comment (dholt) [reply playbooks/ngc-ready.yml]: ok, done

**Pull request merged**